sites/public/src/tsx_content/get-started-cards.tsx

```diff
@@ -8,113 +8,126 @@ import {
 } from "@bloom-housing/doorway-ui-components"
 
 export const getStartedCardIntro: React.ReactElement<CardProps> = (
-  <Card className="border-0 p-0">
-    <Card.Section>
-      GET STARTED Lorem ipsum dolor sit amet consectetur adipisicing elit. Magnam ullam a sunt
-      veniam officiis quae vitae similique non odio, minus minima nisi voluptatem? Sequi veritatis,
-      sunt cumque delectus culpa harum? Lorem ipsum, dolor sit amet consectetur adipisicing elit.
-      Esse illo ullam nulla possimus, incidunt mollitia culpa quam, ex sequi totam provident iusto.
-      Velit totam deleniti unde fugiat minima omnis commodi! Lorem ipsum dolor sit amet consectetur
-      adipisicing elit. Voluptatem excepturi nemo doloribus est consequatur praesentium enim
-      voluptate fuga qui earum repellendus, placeat facere debitis ex eaque repudiandae provident
-      inventore illo.
-    </Card.Section>
-  </Card>
+  <Card className="border-0 p-0"></Card>
```

Comment:
Running into this error, when I build.
![image](https://github.com/metrotranscom/doorway/assets/12400715/47ac56db-b804-40d8-930d-1c0bb6bd8f33)